ui/DesignSystem/Component/StepCounter.svelte

```diff
@@ -1,26 +1,48 @@
 <script>
-  import { Caption } from "../Primitive";
+  import { Icon, Title } from "../Primitive";
 
   export let step = null;
   export let of = null;
```

Comment:
Can we change the props to `selectedStep` and `steps` respectively? In hindsight `of` sounds kinda wrong. Especially when we use it further down in the `each` block.